src/index.html

```diff
@@ -6,9 +6,46 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="navigation">
+        <a href="moyo" class="logo is-active">
+          <img src="images/logo.png" alt="moyo">
+        </a>
+        <ul class="menu">
+          <li class="menu-item">
+            <a href="Apple" class="menu-link is-active">Apple</a>
+          </li>
+          <li class="menu-item">
+            <a href="Samsung" class="menu-link">Samsung</a>
+          </li>
```

Comment:
```suggestion
          <li class="menu-item">
            <a href="Apple" class="menu-link is-active">Apple</a>
          </li>
          
          <li class="menu-item">
            <a href="Samsung" class="menu-link">Samsung</a>
          </li>
```
Add empty line between multiple sibling blocks of html